app/assets/javascripts/app/form-validation.js

```diff
@@ -21,7 +20,6 @@ const validate = {
   },
 
   addEvents() {
-    this.form.addEventListener('change', e => validateField(e.target));
```

Comment:
Brian and I resolved to remove this. See https://github.com/18F/identity-idp/pull/583#issuecomment-255213724 as ref